CMakeLists.txt

```diff
@@ -549,6 +567,45 @@ if(INSTALL_TOOLS)
       )
 endif()
 
+if(COMPILE_EXAMPLES AND QNX)
+    install(DIRECTORY ${PROJECT_BINARY_DIR}/examples/
+      DESTINATION bin/Fast-DDS_examples
+      PATTERN "*.cmake" EXCLUDE
+      PATTERN "*.d" EXCLUDE
+      PATTERN "*.dir" EXCLUDE
+      PATTERN "*.internal" EXCLUDE
+      PATTERN "*.make" EXCLUDE
+      PATTERN "*.marks" EXCLUDE
+      PATTERN "*.o" EXCLUDE
+      PATTERN "*.ts" EXCLUDE
+      PATTERN "*.txt" EXCLUDE
+      PATTERN "CMakeFiles" EXCLUDE
+      PATTERN "Makefile" EXCLUDE
+      PATTERN "cmake" EXCLUDE
+      )
+endif()
+
+if(EPROSIMA_BUILD_TESTS AND QNX)
+    install(DIRECTORY ${PROJECT_BINARY_DIR}/test/
+      DESTINATION bin/Fast-DDS_test
+      PATTERN "*.cmake" EXCLUDE
+      PATTERN "*.d" EXCLUDE
+      PATTERN "*.dir" EXCLUDE
+      PATTERN "*.internal" EXCLUDE
+      PATTERN "*.make" EXCLUDE
+      PATTERN "*.marks" EXCLUDE
+      PATTERN "*.o" EXCLUDE
+      PATTERN "*.ts" EXCLUDE
+      PATTERN "*.txt" EXCLUDE
+      PATTERN "CMakeFiles" EXCLUDE
+      PATTERN "Makefile" EXCLUDE
+      PATTERN "cmake" EXCLUDE
+      )
+    install(DIRECTORY ${PROJECT_SOURCE_DIR}/test/certs
+      DESTINATION bin/Fast-DDS_test
+    )
+endif()
```

Comment:
I install them directly because the test binaries compiled for QNX can only be run on QNX targets, not on your Ubuntu build machine.

I moved it to the CMakeLists.txt inside the test subfolder.